command/subcommand.go

```diff
@@ -50,31 +50,29 @@ func ResolveSubcommand(args []string, config *Config) (bool, *kubectl.Subcommand
 	return subcommandFound && isColoringSupported(subcommandInfo.Subcommand), subcommandInfo
 }
 
-func isColoringSupported(sc kubectl.Subcommand) bool {
-	// when you add something here, it won't be colorized
-	unsupported := []kubectl.Subcommand{
-		kubectl.Create,
-		kubectl.Delete,
-		kubectl.Edit,
-		kubectl.Attach,
-		kubectl.Replace,
-		kubectl.Completion,
-		kubectl.Exec,
-		kubectl.Proxy,
-		kubectl.Plugin,
-		kubectl.Wait,
-		kubectl.Run,
-		kubectl.Ctx,
-		kubectl.Ns,
-		kubectl.Debug,
-		kubectl.KubectlPlugin,
-	}
+// when you add something here, it won't be colorized
+var unsupported = map[kubectl.Subcommand]struct{}{
+	kubectl.Attach:        {},
+	kubectl.Completion:    {},
+	kubectl.Create:        {},
+	kubectl.Ctx:           {},
+	kubectl.Debug:         {},
+	kubectl.Delete:        {},
+	kubectl.Edit:          {},
+	kubectl.Exec:          {},
+	kubectl.KubectlPlugin: {},
+	kubectl.Ns:            {},
+	kubectl.Plugin:        {},
+	kubectl.Proxy:         {},
+	kubectl.Replace:       {},
+	kubectl.Run:           {},
+	kubectl.Wait:          {},
 
-	for _, u := range unsupported {
-		if sc == u {
-			return false
-		}
-	}
+	// oc (OpenShift CLI) specific subcommands
+	kubectl.Rsh: {},
+}
 
-	return true
+func isColoringSupported(sc kubectl.Subcommand) bool {
+	_, found := unsupported[sc]
+	return !found
 }
```

Comment:
Changing this from a slice to a map does have performance implications.

When benchmarking, a slice was as expected faster when looking up the first element in the list, but the map approach was actually faster when looking up the last element.

```console
$ go test -bench=. ./command -benchtime=5s
goos: linux
goarch: amd64
pkg: github.com/kubecolor/kubecolor/command
cpu: 11th Gen Intel(R) Core(TM) i7-1165G7 @ 2.80GHz
BenchmarkIsColoringSupported_rsh-8         	489333993	        16.41 ns/op
BenchmarkIsColoringSupportedMap_rsh-8      	531382500	        13.78 ns/op
BenchmarkIsColoringSupported_attach-8      	1000000000	         4.193 ns/op
BenchmarkIsColoringSupportedMap_attach-8   	680426997	         9.031 ns/op
PASS
ok  	github.com/kubecolor/kubecolor/command	29.554s
```

However the code is way smaller and in my opinion the benefit of the clean code wins over the cost of some nanoseconds (btw a nanosecond is 0.000000001 seconds)